plugin/src/main/java/org/craftercms/commons/plugin/model/Plugin.java

```diff
@@ -209,4 +213,39 @@ public void setSearchEngine(final String searchEngine) {
         this.searchEngine = searchEngine;
     }
 
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (!(o instanceof Plugin)) {
+            return false;
+        }
+        final Plugin plugin = (Plugin)o;
+        return Objects.equals(type, plugin.type) && Objects.equals(id, plugin.id) && Objects.equals(name,
```

Comment:
Should it be this big? Isn't it enough with comparing the `id` and `type` maybe?

Reply:
we could make it use only `id`, `type` and `version` but the main reason I added all fields is to quickly compare in unit tests without having to list all fields there